src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.handlers.DukeCommand;
+import duke.models.Parser;
+import duke.models.Storage;
+import duke.models.TaskList;
+import duke.models.Ui;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static TaskList taskList;
```

Comment:
Perhaps taskList should not be a static variable?
```suggestion
    public TaskList taskList;
```